user-guide/Advanced_Modules/DOM/DomHelper_class.md

```diff
@@ -54,6 +57,46 @@ if (!traceData.HasSucceeded())
 }
 ```
 
+### Multiple instances
+
+When multiple `DomInstances` need to get created, updated, or deleted, we recommend calling the *CreateOrUpdate* or *Delete* methods on a `DomInstance` CRUD helper component with a list of those `DomInstances`. This feature is available from DataMiner 10.4.2/10.5.0 onwards<!-- RN 37891 -->.
+
+For example, to create or update multiple `DomInstances`:
+
+```csharp
+// Create or update a list of DomInstances
+var createResult = helper.DomInstances.CreateOrUpdate(domInstances);
+```
+
+*CreateOrUpdate* will consider `DomInstances` that are stored already as `DomInstances` that need an update. The remaining `DomInstances` will be created. A mix of both is allowed.
```

Comment:
I had to read this sentence a couple of times, before it became clear. Is it correct that the main message is that if you provide DomInstances that already exist that these will be considered as an update? And if they don't exist yet, that they will be created?

Reply:
```suggestion
*CreateOrUpdate* will consider the provided `DomInstances` that already exist as updates, the other `DomInstances` will be created. It is supported to provide a mix of both.
```